src/containers/shared/components/Transaction/PaymentChannelClaim/parser.ts

```diff
@@ -1,16 +1,20 @@
-const formatAmount = require('./formatAmount')
+import { Meta } from '../../../transactionUtils'
```

Comment:
Should this be a type import? (Ditto for the other types you exported from the `transactionUtils` file)

Reply:
Sure why not.